src/ml_board/frontend/newDashboard/src/websocket/SocketClass.tsx

```diff
@@ -0,0 +1,166 @@
+import { io } from 'socket.io-client';
+import EVENT_TYPE from './socketEventConstants';
+import handleEvaluationResultData from './event_handlers/evaluationResultDataHandler';
```

Comment:
is this really defined in `evaluationResultDataHandler`?

Reply:
Removed from here and used this method in worker_utils.tsx
Also defined method with this name in  evaluationResultDataHandler (before, there was some issue in naming)